 lib/Froxlor/Froxlor.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Froxlor/Froxlor.php b/lib/Froxlor/Froxlor.php
index a7f0c7a3..bc286bd1 100644
--- a/lib/Froxlor/Froxlor.php
+++ b/lib/Froxlor/Froxlor.php
@@ -13,7 +13,7 @@ final class Froxlor
 	const DBVERSION = '201910120';
 
 	// Distribution branding-tag (used for Debian etc.)
-	const BRANDING = '';
+	const BRANDING = '-1';
 
 	/**
 	 * return path to where froxlor is installed, e.g.
