--- a/lib/functions.php
+++ b/lib/functions.php
@@ -61,6 +61,13 @@
 				return;
 			}
 
+			// added for using the system copy of PHPMailer
+			if ($classname == 'PHPMailer' || $classname == 'SMTP')
+			{
+				include '/usr/share/php/libphp-phpmailer/' . strtolower($filename);
+				return;
+			}
+
 			if(is_dir($dirname . $filename))
 			{
 				findIncludeClass($dirname . $filename . '/', $classname);
--- a/scripts/jobs/cron_traffic_report.php
+++ b/scripts/jobs/cron_traffic_report.php
@@ -24,7 +24,7 @@
  * Initialize the mailingsystem
  */
 
-require (dirname(__FILE__) . '/../lib/class.phpmailer.php');
+require '/usr/share/php/libphp-phpmailer/class.phpmailer.php';
 $mail = new PHPMailer();
 $mail->SetFrom($settings['panel']['adminmail'], 'Froxlor Administrator');
 
