From: <elacour@home-dn.net>
Subject: Update configuration file to debian layout

--- ./sample/linux/libnss-mysql.cfg	2005-01-31 03:16:46.000000000 +0100
+++ ./sample/linux/libnss-mysql.cfg	2008-11-05 14:43:20.000000000 +0100
@@ -35,6 +35,6 @@
 database    auth
 username    nss-user
 password    userpass
-#socket      /var/lib/mysql/mysql.sock
+#socket      /var/run/mysqld/mysqld.sock
 #port        3306
 
