--- a/templates/misc/configfiles/debian_wheezy/postfix_courier/etc_postfix_main.cf
+++ b/templates/misc/configfiles/debian_wheezy/postfix_courier/etc_postfix_main.cf
@@ -688,9 +688,10 @@ smtpd_sender_restrictions = permit_mynetworks,
 smtpd_client_restrictions = permit_mynetworks,
 	permit_sasl_authenticated,
 	reject_unknown_client_hostname
-smtpd_relay_restrictions = permit_mynetworks,
-	permit_sasl_authenticated,
-	defer_unauth_destination
+
+# Postfix 2.10 requires this option. Postfix < 2.10 ignores this.
+# The option is intentionally left empty.
+smtpd_relay_restrictions = 
 
 # Maximum size of Message in bytes (50MB)
 message_size_limit = 52428800
diff --git a/templates/misc/configfiles/debian_wheezy/postfix_dovecot/etc_postfix_main.cf b/templates/misc/configfiles/debian_wheezy/postfix_dovecot/etc_postfix_main.cf
index b797bed..4e83c37 100644
--- a/templates/misc/configfiles/debian_wheezy/postfix_dovecot/etc_postfix_main.cf
+++ b/templates/misc/configfiles/debian_wheezy/postfix_dovecot/etc_postfix_main.cf
@@ -688,9 +688,10 @@ smtpd_sender_restrictions = permit_mynetworks,
 smtpd_client_restrictions = permit_mynetworks,
 	permit_sasl_authenticated,
 	reject_unknown_client_hostname
-smtpd_relay_restrictions = permit_mynetworks,
-	permit_sasl_authenticated,
-	defer_unauth_destination
+
+# Postfix 2.10 requires this option. Postfix < 2.10 ignores this.
+# The option is intentionally left empty.
+smtpd_relay_restrictions = 
 
 # Maximum size of Message in bytes (50MB)
 message_size_limit = 52428800
